sky/clouds/cloud.py

```diff
@@ -66,6 +68,14 @@ class Cloud:
 
     _REPR = '<Cloud>'
 
+    _MAX_CLUSTER_NAME_LEN_LIMIT: Optional[int] = None
+
+    # All features implemented by this cloud. Used by
+    # Cloud.check_features_are_supported()
+    # None means all features are supported.
+    _CLOUD_IMPLEMENTATION_FEATURES: Optional[
+        Set[CloudImplementationFeatures]] = None
```

Comment:
For discussion: I feel putting these as methods will more clearly remind implementors that they need to be filled in. Here, the naming of the vars suggests they are "constants", which may be neglected?